Library/Homebrew/test/test_keg.rb

```diff
@@ -304,4 +304,35 @@ def test_removes_broken_symlinks_that_conflict_with_directories
     keg.unlink
     keg.uninstall
   end
+
+  def test_mach_o_files_skips_hardlinks
+    a = (HOMEBREW_CELLAR/"a")/"1.0"
```

Comment:
Yep, I wasn't thinking. I'll fix this now.
Funny that Ruby 1.8.7 chokes on that whitespace ambiguity :smile: